content/en/blog/_posts/2022-12-01-cpumanager-goes-GA.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: blog
+title: 'Kubernetes 1.26: CPUManager goes GA'
```

Comment:
nit:
```suggestion
title: 'Kubernetes v1.26: CPUManager goes GA'
```

Reply:
thanks, fixed (all instances of versions)